docker/webhook.Dockerfile

```diff
@@ -1,5 +1,5 @@
-ARG BUILDER=mcr.microsoft.com/oss/go/microsoft/golang:1.21-bullseye
```

Comment:
are we using these as args anywhere? if not, let's add them directly and have dependabot manage these (applies to all)

Reply:
done!